Resources/Prototypes/Entities/Objects/Consumable/Food/taco.yml

```
@@ -166,6 +166,33 @@
         - ReagentId: Vitamin
           Quantity: 6
 
+- type: entity
+  name: draco
+  parent: FoodTacoBase
+  id: FoodTacoDragon
+  description: A dragon taco, which is technically also a fish taco.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - meaty
+      - spicy
```

```suggestion
    flavors:
    - meaty
    - spicy
```
[yaml convention](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions)